api/presenter/job.go

```diff
@@ -1,6 +1,19 @@
 package presenter
 
-import "net/url"
+import (
+	"fmt"
+	"net/url"
+)
+
+const (
+	JobGUIDDelimiter = "~"
```

Comment:
Reasons why I chose `~`:
https://github.com/cloudfoundry/korifi/issues/1116#issuecomment-1142327974

Happy to consider other options as well.